CodeGen/templates/Action.h

```diff
@@ -3,30 +3,43 @@
 
 #pragma once
 
+// UE
 #include <CoreMinimal.h>
 
+// ROS
 #include "unique_identifier_msgs/msg/uuid.h"
 #include "{{data.Group}}/action/{{data.Name}}.h"
 #include "action_msgs/srv/cancel_goal.h"
 
+// rclUE
 #include "Actions/ROS2GenericAction.h"
 #include "rclcUtilities.h"
 
-#include "ROS2{{data.NameCap}}Action.generated.h"
+
+// Generated Msg/Srv/Action(can be empty)
+{{data.GoalHeaders}}
+{{data.ResultHeaders}}
+{{data.FeedbackHeaders}}
+
+// Generated
+#include "ROS2{{data.UEName}}.generated.h"
 
 USTRUCT(Blueprintable)
-struct {{data.ModuleAPI}} F{{data.StructName}}SendGoalRequest
+struct {{data.ModuleAPI}} FROS{{data.UEName}}SGReq
 {
 	GENERATED_BODY()
 
 public:
-  	TArray<uint, TFixedAllocator<16>> GoalId;
+	{{data.GoalConstantsDef}}
+  	
+	TArray<uint, TFixedAllocator<16>> GoalId;
 
 	{{data.GoalTypes}}
 
-	F{{data.StructName}}SendGoalRequest()
+	FROS{{data.UEName}}SGReq()
     {
         UROS2Utils::GenerateRandomUUID16(GoalId);
+		{{data.GoalConstructor}}
```

Comment:
Could it be in initialization list like `FROS{{data.UEName}}SGReq():{{data.GoalConstructor}}?`

Reply:
it is not constructor call but initialization step such as. I added this to initialize fixed size array
![image](https://user-images.githubusercontent.com/16463135/195031814-852af3db-b323-4180-960d-2ee4601c85db.png)

Reply:
I see.., btw, should it be `Covariance.SetNumZeroed(36);` only, without for loop?